app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginFragment.java

```diff
@@ -109,16 +111,22 @@ private void saveLoginDataOnSharedPreferences(String email, String password) {
 
     @Override
     public void onInvalidLoginData() {
-        //TODO: toask message "Incorrect email or password"
+        Toast toast = Toast.makeText(getContext(), getString(
+                R.string.fragment_login_invalid_email_or_password_toast), Toast.LENGTH_LONG);
+        toast.show();
     }
 
     @Override
     public void onServerError() {
-        //TODO: toast message "Sorry, (bla bla) try again later"
+        Toast toast = Toast.makeText(getContext(), getString(
+                R.string.fragment_login_server_error_toast), Toast.LENGTH_LONG);
+        toast.show();
     }
 
     @Override
     public void onInvalidCallError() {
```

Comment:
Maybe we can check network connection before the toast. The current message was only for testing the different callbacks